interface/patient_file/report/custom_report.php

```diff
@@ -222,751 +222,755 @@ function zip_content($source, $destination, $content = '', $create = true)
-                   " left join code_types ct on ct.ct_key = 'CVX' " .
-                   " left join codes c on c.code_type = ct.ct_id AND i1.cvx_code = c.code " .
-                   " where i1.patient_id = ? and i1.added_erroneously = 0 " .
-                   " order by administered_date desc";
-                $result = sqlStatement($sql, array($pid));
-                while ($row = sqlFetchArray($result)) {
-                  // Figure out which name to use (ie. from cvx list or from the custom list)
-                    if ($GLOBALS['use_custom_immun_list']) {
-                         $vaccine_display = generate_display_field(array('data_type' => '1','list_id' => 'immunizations'), $row['immunization_id']);
-                    } else {
-                        if (!empty($row['code_text_short'])) {
-                            $vaccine_display = xlt($row['code_text_short']);
+                            echo "<tr><td>&nbsp;</td></tr>";
+                            echo "<tr><td class=bold>" . xlt('Sub-Total') . "</td><td class=text>" . text(oeFormatMoney($total + abs($copays))) . "</td></tr>";
+                            echo "<tr><td class=bold>" . xlt('Paid') . "</td><td class=text>" . text(oeFormatMoney(abs($copays))) . "</td></tr>";
+                            echo "<tr><td class=bold>" . xlt('Total') . "</td><td class=text>" . text(oeFormatMoney($total)) . "</td></tr>";
+                            echo "</table>";
+                            echo "<pre>";
+                            //print_r($billings);
+                            echo "</pre>";
                         } else {
-                            $vaccine_display = generate_display_field(array('data_type' => '1','list_id' => 'immunizations'), $row['immunization_id']);
+                            printPatientBilling($pid);
                         }
-                    }
-
-                    echo text($row['administered_date']) . " - " . $vaccine_display;
-                    if ($row['immunization_note']) {
-                         echo " - " . text($row['immunization_note']);
-                    }
 
-                    echo "<br />\n";
-                }
-
-                echo "</div>\n";
-            }
+                        echo "</div>\n"; // end of billing DIV
+
+                /****
+
+                    } elseif ($val == "allergies") {
+
+                        print "<span class=bold>Patient Allergies:</span><br />";
+                        printListData($pid, "allergy", "1");
+
+                    } elseif ($val == "medications") {
+
+                        print "<span class=bold>Patient Medications:</span><br />";
+                        printListData($pid, "medication", "1");
+
+                    } elseif ($val == "medical_problems") {
+
+                        print "<span class=bold>Patient Medical Problems:</span><br />";
+                        printListData($pid, "medical_problem", "1");
+
+                ****/
```

Comment:
Here contains some commented code, should I remove it?

Reply:
yep, feel free to remove commented out code :)

Reply:
OK, remove it :)